### ISSUE TITLE
git-gui broken after upgrade to tcl-tk 9.0

### ISSUE THREAD

**Issue by stkain** [Homebrew/homebrew-core#197304]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
After "brew update" "brew upgrade", tcl-tk 9.0 was installed.

Both gitk and git gui do not work anymore after todays upgrade of tcl-tk to version 9.0
```

### Verification

- [X] My `brew doctor` output says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

run gitk after "brew update", "brew upgrade"

### What happened (include all command output)?

Popup is displayed: "Sorry, gitk cannot run with this version of Tcl/TK. Gitk requires at least Tcl/TK 8.4

### What did you expect to happen?

Normal startup.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
git update
git upgrade
gitk
```

**Comment by stkain** [comment]
brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

**Comment by ZhongRuoyu** [comment]
This should be fixed by #197299. Please run `brew update` and `brew upgrade`, then try again.

**Comment by carcinocron** [comment]
Any solution for this that works in 2025? According to https://formulae.brew.sh/formula/git-gui the latest is back to 9 and:

<img width="295" height="184" alt="Image" src="https://github.com/user-attachments/assets/8165470c-ffda-421d-90b9-2ba64e448926" />
